apps/admin-gui/src/app/vos/pages/group-detail-page/group-applications/group-applications.component.html

```diff
@@ -44,12 +36,24 @@ <h1 class="page-subtitle">{{'VO_DETAIL.APPLICATION.TITLE' | translate}}</h1>
     [placeholder]="'VO_DETAIL.APPLICATION.FILTER'"></perun-web-apps-debounce-filter>
   <perun-web-apps-date-range (datePicker)="select()" [endDate]="endDate" [startDate]="startDate">
   </perun-web-apps-date-range>
+  <div>
+    <button
+      mat-stroked-button
+      *ngIf="columnsAuth"
+      class="ml-2"
+      [disabled]="showAllDetails"
```

Comment:
can we have a tooltip saying why is the option disabled when all details are shown?